src/compositor/output.c

```diff
@@ -234,6 +234,44 @@ finish_frame_tasks(struct wlc_output *output)
 }
 
 static void
+render_subsurface(struct wlc_output *output, struct wlc_surface *surface, struct wlc_point offset, struct wlc_coordinate_scale parent_scale) {
+   struct wlc_geometry g = (struct wlc_geometry) {
+       .origin = {offset.x + parent_scale.w * (surface->commit.subsurface_position.x + surface->commit.offset.x),
+                  offset.y + parent_scale.h * (surface->commit.subsurface_position.y + surface->commit.offset.y)},
+       .size = surface->size
+   };
+
+   wlc_render_surface_paint(&output->render, &output->context, surface, &g);
+}
+
+static void
+subsurfaces_render(struct wlc_output *output, struct wlc_surface *surface, struct wlc_coordinate_scale parent_scale, struct chck_iter_pool *callbacks, struct wlc_point offset) {
+
+   if(!surface)
```

Comment:
I think Cloudef means adding an space between the if and the parenthesis, like this: if (!surface)